src/test/java/racingcar/CarTest.java

```diff
@@ -2,42 +2,66 @@
 
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import racingcar.model.Car;
+import racingcar.model.CarName;
+import racingcar.model.Distance;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 @DisplayName("Car 클래스 테스트")
 public class CarTest {
 
+    private static final CarName TEST_CAR_NAME = CarName.from("test");
+
+    @Test
+    @DisplayName("Car 생성시 CarName 에 null 이 들어오면 오류가 발생한다.")
+    void carNameNullTest() {
+        assertThatThrownBy(() -> new Car(null, Distance.MOVE_STATE))
+                .isInstanceOf(NullPointerException.class)
+                .hasMessageContaining("name is null");
+    }
+    @Test
+    @DisplayName("Car 생성시 state 에 null 이 들어오면 오류가 발생한다.")
+    void carStateNullTest() {
+        assertThatThrownBy(() -> new Car(TEST_CAR_NAME, null))
+                .isInstanceOf(NullPointerException.class)
+                .hasMessageContaining("state is null");
+    }
+
     @Test
     @DisplayName("자동차의 초기 상태는 0이어야 한다.")
     void carStateTest() {
-        Car car = new Car();
-        assertThat(car.state()).isEqualTo(PositiveInteger.ZERO);
+        Car car = new Car(TEST_CAR_NAME);
+
+        assertThat(car.state()).isEqualTo(Distance.ZERO);
+    }
+
+    @Test
+    @DisplayName("자동차 생성시 지정한 이름은 name()을 통해서 얻을 수 있다.")
+    void carNameTest() {
+        Car car = new Car(TEST_CAR_NAME);
+
+        assertThat(car.name().value()).isEqualTo(TEST_CAR_NAME.value());
```

Comment:
equals(), hashCode()를 잘 구현해주셔서 다음과 같이 동등성 비교가 가능할 수 있겠네요. 😄 
```suggestion
        assertThat(car.name()).isEqualTo(TEST_CAR_NAME);
```

Reply:
앗 그러네요! 오버라이딩, 테스트로 충분히 확인했으니 value()생략하고 비교하겠습니다 감사합니다 ㅎㅎ